add-on/_locales/en/messages.json

```diff
@@ -63,13 +63,13 @@
     "message": "Active Tab",
     "description": "A menu item in Browser Action pop-up (panel_activeTabSiteRedirectEnable)"
   },
-  "panel_activeTabSiteRedirectToggle": {
-    "message": "Redirect on $1",
-    "description": "A menu item in Browser Action pop-up (panel_activeTabSiteRedirectToggle)"
+  "panel_activeTabSiteIntegrationsToggle": {
+    "message": "Enable on $1",
```

Comment:
Not sure if `Enable on <hostname>` is the best label for this, but it is short and simple.
Let me know if there is a better one.